### PR TITLE
ADD: Colorbar Manipulation Example

### LINKED ISSUE #1283: DOC: Add an example showing how to modify the colorbar
It would be helpful to have an example that shows how to extract the matplotlib colorbar from the display so the user can modify the size, font, etc.

We show how to do this in the [SAIL example](https://arm-development.github.io/sail-xprecip-radar/radar-precip/plot-method-description-figure.html), but this should be in our main documentation.

### PR DESCRIPTION
**Pull request by jrobrien91**
Example to manipulate the title and size of a colorbar object 

Closes #1283

### CODE REVIEW

File: examples/plotting/plot_modify_colorbar.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+# Make sure labels are only plotted on the left and bottom
+gl.top_labels = False
+gl.right_labels = False
+
+gl.xlabel_style = {'size': 12}
+gl.ylabel_style = {'size': 12}
+
+# Define the colorbar from the RadarMapDisplay object
+#Modify the colorbar title and size
+cbar = plt.colorbar(mappable=display.plots[0], fraction=0.04, ax=axsB)
```
Comment (mgrover1) [inline examples/plotting/plot_modify_colorbar.py:116]: Should you be able to set cbar = display.cbs[0]? This is usually how I modify it.. display.cbs should return a list of colorbar objects

**Review by zssherman — CHANGES_REQUESTED** [review]

**Comment by zssherman** [comment]
@jrobrien91 Figured out why it wasn't showing up, my comments were put into pending and I had to start a review to get them to appear.

File: examples/plotting/plot_modify_colorbar.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+# Define and Read in the test data
+radar_file = get_test_data('swx_20120520_0641.nc')
+radar = pyart.io.read(radar_file)
+
+# Create the Radar Map Display (defines x,y as lat/lons)
+display = pyart.graph.RadarMapDisplay(radar)
+
+# Display the horizontal equivalent reflectivity factor
+# Note: embellish = False will remove initial lat/lon coords
+display.plot_ppi_map('reflectivity_horizontal',
```
[the remark this answers is not in the document]
Comment (mgrover1) [reply examples/plotting/plot_modify_colorbar.py:46]: I think we should stick with the multiple lines... It makes it easier to read, and conforms to the same standards we are using ACT

**Comment by jrobrien91** [comment]
Should have some time to work on this today. Sorry for the delay!

**Comment by mgrover1** [comment]
@jrobrien91 do you need any help getting this to the finish line? Happy to help!

File: examples/plotting/plot_modify_colorbar.py [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+del display
+
+############################################
+# ** Colorbar Position / Title Manipulation
+
+# Now, let's update the colorbar position
+# to match the display
+
+# Create the Radar Map Display (defines x,y as lat/lons)
+#display = pyart.graph.RadarMapDisplay(radar)
```
Comment (zssherman) [inline examples/plotting/plot_modify_colorbar.py:82]: Can we remove the unused code?

**Review by mgrover1 — APPROVED** [review]
Thanks for making those changes @jrobrien91

**Review by zssherman — APPROVED** [review]

File: examples/plotting/plot_modify_colorbar.py [hunk 4]
```diff
@@ -0,0 +1,123 @@
+"""
+=================================
+Modifiy a Colorbar for your Plot
```
Comment (zssherman) [inline examples/plotting/plot_modify_colorbar.py:3]: Sorry last thing, modifiy to modify

Comment (zssherman) [reply examples/plotting/plot_modify_colorbar.py:3]: @jrobrien91 Sorry one last thing, beside that looks good!

**Comment by zssherman** [comment]
Thanks @jrobrien91 merging.

**Pull request merged**